Examples/basic_examples/kinematics_basic_demo.py

```
@@ -220,22 +230,25 @@ def demonstrate_inverse_kinematics(self):
                 print(f"   Solution: {solution}")
                 
-                # Verify the solution
+                # Verify the solution in task space (FK of the IK solution vs. target).
                 verification_pose = self.robot.forward_kinematics(solution)
                 position_error = np.linalg.norm(verification_pose[:3, 3] - target_position)
                 orientation_error = self._compute_orientation_error(verification_pose[:3, :3], target_pose[:3, :3])
-                
+                # Joint-space residual against the configuration that produced the target.
+                joint_error = float(np.linalg.norm(solution - fk_result['joint_angles']))
+
                 print(f"   Verification:")
```

Ruff (0.15.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)